README.rst

```diff
@@ -1,7 +1,7 @@
 .. raw:: html
 
    <p align="center">
-     <img width="450" height="200" src=docs/tialab_logo.png>
+     <img width="450" height="200" src=https://warwick.ac.uk/fac/sci/dcs/research/tia/models/tialab_logo.png>
```

Comment:
seems fine, but we should be careful about using this page as a dumping ground. Would be good to have some more organisation in terms of where files / models will be stored.

Reply:
Created a subpage to restructure and updated in https://github.com/TIA-Lab/tiatoolbox/commit/9cbb54bd38131dbfcbe2b4e54a0da5dd46edf77c